libdnf/conf/config_parser.cpp

```diff
@@ -42,12 +45,17 @@ void ConfigParser::substitute(std::string & text, const std::map<std::string, st
         } else {
             bracket = false;
         }
+        auto variable1 = static_cast<signedType>(variable);
```

Comment:
It might be good to rename `variable1` to `variable_signed` or something similar that suggests the purpose or the content.